manifests/implementation/mattermost/mattermost-team-edition/install.yaml

```diff
@@ -499,3 +509,26 @@ spec:
               artifacts:
               - name: merged
                 path: /merged.yaml
+
+          - name: resolve-ti-art-value
+            inputs:
+              artifacts:
+                - name: ti-artifact
+                  path: /tmp/input-ti.yaml
+                - name: backend
+                  path: /tmp/storage-backend.yaml
+            outputs:
+              artifacts:
+                - name: ti-artifact
+                  path: /tmp/output.yaml
+            container:
+              image: ghcr.io/capactio/pr/ti-value-fetcher:PR-697
+              env:
+                - name: APP_LOGGER_DEV_MODE
+                  value: "true"
```

Comment:
```suggestion
```

Reply:
I would keep it as it is useful to have additional logging, especially in workflows. We also have logger dev mode enabled in runners, e.g. Helm runner, so it's consistent 🙂